misc/multistream-select/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.12.0 [Unreleased]
+
+- Remove parallel dialing optimization, to avoid requiring the use of the `ls` command.
```

Comment:
```suggestion
- Remove parallel dialing optimization, to avoid requiring the use of the `ls` command. See [PR 2934].

[PR 2934]: https://github.com/libp2p/rust-libp2p/pull/2934
```